### PR TITLE
Hello interpreter fixes

### PR DESCRIPTION
**Pull request by aszlam**
# Description

Gets Look and Move mostly working from language-> full agent pipeline on hello_robot

## Type of change

Please check the options that are relevant.

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [x] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change
- [ ] Datasets Release
- [ ] Models Release

## Type of requested review

- [ ] I want a thorough review of the implementation.
- [x] I want a high level review. 
- [ ] I want a deep design review.

## Before and After

Previously there was lots of older interpreter code that had not updated along the craftassist side.  This is fixed here, albeit with several less-than-high quality shortcuts to get the system stood up.  My plan is to merge these, connect to @soumith's upcoming PRs with explore, and do lots of on robot testing and cleanup.   

# Testing

Did a good deal of on robot testing; there are still some rough edges w.r.t. coordinate transforms

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [x] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [x] New and existing unit tests pass locally with my changes.
- [ ] I have added relevant collaborators to review the PR before merge.
- [ ] [Polymetis only] I have verified that all scripts in `tests/scripts` runs on hardware.

### CODE REVIEW

**Comment by aszlam** [comment]
for review:  I don't actually want detailed review in tasks/interpreter; a lot of that is ugly and is going to evolve over next few PRs (especially w.r.t. rotations).  rn I am just trying to get the full pipeline working on robot, then will clean up and make more correct.   otoh I don't want to introduce bugs that mess up @anuragprat1k even in short term, so can y'all focus on hello_robot_mover.py and the other changes to droidlet/lowlevel?

File: droidlet/lowlevel/hello_robot/hello_robot_mover.py [hunk 1]
```diff
@@ -14,18 +14,17 @@
 
 import cv2
 from droidlet.shared_data_structs import ErrorWithResponse
-from agents.argument_parser import ArgumentParser
+#from agents.argument_parser import ArgumentParser
```
Comment (anuragprat1k) [inline droidlet/lowlevel/hello_robot/hello_robot_mover.py:17]: this is a modified version of `argparse.ArgumentParser`. Use this instead?

File: droidlet/lowlevel/hello_robot/hello_robot_mover.py [hunk 2]
```diff
@@ -87,12 +89,67 @@ def bot_step(self):
         f = not self.bot.is_moving().value
         return f
 
-    def look_at(self, target, turn_base=True):
+    def relative_pan_tilt(self, dpan, dtilt, turn_base=True):
```
Comment (anuragprat1k) [inline droidlet/lowlevel/hello_robot/hello_robot_mover.py:92]: follow the same naming conventions throughout of naming variables as `*_rad` if they are in radians?

File: droidlet/lowlevel/hello_robot/hello_robot_mover.py [hunk 3]
```diff
@@ -110,26 +167,20 @@ def look_at(self, target, turn_base=True):
         logging.info(f"looking at x,y,z: {target}")
         
         pan_rad, tilt_rad = get_camera_angles(cam_pos, target)
+#        pan_res = angle_diff(pos[2], pan_rad)
         # For the Hello camera, negative tilt seems to be up, and positive tilt is down
         # For the locobot camera, it is the opposite
         # TODO: debug this further, and make things across robots consistent
-        tilt_rad = -tilt_rad
+        #tilt_rad = -tilt_rad
```
Comment (anuragprat1k) [inline droidlet/lowlevel/hello_robot/hello_robot_mover.py:174]: nit: generally just delete code that you don't need instead of commenting them

File: droidlet/lowlevel/hello_robot/hello_robot_mover.py [hunk 4]
```diff
@@ -182,28 +233,27 @@ def move_absolute(self, xyt_positions, blocking=True):
...
-         return:
-         (x, z, yaw) of the robot base in standard coordinates
+        return:
+        (x, z, yaw) of the robot base in standard coordinates
         """
         future = safe_call(self.bot.get_base_state)
         x_global, y_global, yaw = future.value
-        x_standard = -y_global
-        z_standard = x_global
+        x_standard = x_global
```
Comment (anuragprat1k) [inline droidlet/lowlevel/hello_robot/hello_robot_mover.py:255]: So do we have a different standard coordinate system for hello? Why is that necessary?

File: droidlet/tools/artifact_scripts/tracked_checksums/datasets.txt [hunk 5]
```diff
@@ -1 +1 @@
-3e7dfe1694860f8339e4ed15ae3e5a491ece2a90
+8559f0495013cd9b4a54f5e86db8b1d3c1f03128
```
Comment (anuragprat1k) [inline droidlet/tools/artifact_scripts/tracked_checksums/datasets.txt:1]: what's changed in the datasets in this PR?

File: droidlet/lowlevel/hello_robot/hello_robot_mover.py (hunk 3 above)
Comment (aszlam) [reply droidlet/lowlevel/hello_robot/hello_robot_mover.py]: done

File: droidlet/lowlevel/hello_robot/hello_robot_mover.py (hunk 2 above)
Comment (aszlam) [reply droidlet/lowlevel/hello_robot/hello_robot_mover.py:94]: can I get a pass on this for now?  would rather just convert everything to radians once everything is working

File: droidlet/lowlevel/hello_robot/hello_robot_mover.py (hunk 1 above)
Comment (aszlam) [reply droidlet/lowlevel/hello_robot/hello_robot_mover.py:17]: don't need the full agent argument parser here, its just for testing the mover interactively from ipython

File: droidlet/tools/artifact_scripts/tracked_checksums/datasets.txt (hunk 5 above)
Comment (aszlam) [reply droidlet/tools/artifact_scripts/tracked_checksums/datasets.txt:1]: some of the hi-pri commands

**Review by anuragprat1k — APPROVED** [review]
